<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsComposerItemWidgetBase</class>
 <widget class="QWidget" name="QgsComposerItemWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QPushButton" name="mPositionButton">
     <property name="text">
      <string>Position and size...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QgsCollapsibleGroupBox" name="mFrameGroupBox"><property name="collapsed" stdset="0"><bool>false</bool></property><property name="saveCollapsedState" stdset="0"><bool>true</bool></property>
     <property name="title">
      <string>Show frame</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QPushButton" name="mFrameColorButton">
        <property name="text">
         <string>Frame color...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="mOutlineWidthLabel">
        <property name="text">
         <string>Thickness</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="buddy">
         <cstring>mOutlineWidthSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="mOutlineWidthSpinBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QgsCollapsibleGroupBox" name="mBackgroundGroupBox"><property name="collapsed" stdset="0"><bool>false</bool></property><property name="saveCollapsedState" stdset="0"><bool>true</bool></property>
     <property name="title">
      <string>Show background</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="3">
       <widget class="QPushButton" name="mBackgroundColorButton">
        <property name="text">
         <string>Background color...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="mOpacityLabel">
        <property name="text">
         <string>Opacity</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="buddy">
         <cstring>mOpacitySlider</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="mOpacitySlider">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="mOpacitySpinBox">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="mIdLabel">
     <property name="text">
      <string>Item ID</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="mItemIdLineEdit"/>
   </item>
   <item row="6" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>143</width>
       <height>87</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>mOpacitySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mOpacitySpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mOpacitySpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mOpacitySlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
